### PR TITLE
Update spot price

### PR DESCRIPTION
**Pull request by vkulinich-cl**
- Update dollar spot price with stableCointRate if the prop is pased
- Added ss58 prefix to XCM

### CODE REVIEW

File: packages/apps/src/app/xcm/App.ts [hunk 1]
```diff
@@ -81,6 +81,7 @@ export class XcmApp extends PoolApp {
   @property({ type: String }) destChain: string = null;
   @property({ type: String }) asset: string = null;
   @property({ type: String }) blacklist: string = null;
+  @property({ type: String }) ss58Prefix: string = null;
```
Comment (nohaapav) [inline packages/apps/src/app/xcm/App.ts:84]: if u use type: Number you don't have to cast to Number down below

**Comment by nohaapav** [comment]
The major problem i see is that if u use different than dollar pegged asset and rate not fetched undefined is used which result to dollar value totally off ..

**Comment by vkulinich-cl** [comment]
> The major problem i see is that if u use different than dollar pegged asset and rate not fetched undefined is used which result to dollar value totally off ..

You mean, I need to check if it's Kusama ecosystem and stablecoin is USDT?

File: packages/apps/src/app/xcm/App.ts (hunk 1 above)
Comment (vkulinich-cl) [reply packages/apps/src/app/xcm/App.ts:84]: ok, didn't know

**Pull request merged**